legacy/firmware/fido2/resident_credential.c

```
@@ -0,0 +1,151 @@
+
+#include "resident_credential.h"
+#include "../i18n/keys.h"
+#include "gettext.h"
+#include "layout2.h"
+#include "se_chip.h"
+
+uint32_t resident_credential_get_count(void) { return 0; }
+
+uint32_t resident_credential_find_by_rp_id_hash(
```

cppcheck (2.10-2)

[style] The function 'resident_credential_find_by_rp_id_hash' is never used.

(unusedFunction)